ods_ci/libs/DataSciencePipelinesKfp.py

```diff
@@ -126,7 +126,7 @@ def create_run_from_pipeline_func(
             pipeline_params["openshift_server"] = self.api.get_openshift_server()
         if "openshift_token" in pipeline_params:
             pipeline_params["openshift_token"] = self.api.get_openshift_token()
-        print(f"pipeline_params modified with dynamic values: {pipeline_params}")
+        print("pipeline_params modified with dynamic values")
```

Comment:
Why this change?

Reply:
The logs were printing the object storage credentials

Reply:
Ok. It was introduced for troubleshooting purpose. Replace by `sorted(pipeline_params.keys())`

Reply:
Done